contracts/HybridCustody.cdc

```diff
@@ -71,6 +69,8 @@ pub contract HybridCustody {
         child: Address,
         pendingParent: Address
     )
+    /// OwnedAccount granted ownership to a new address, publishing a Capability for the pendingOwner
+    pub event OwnershipGranted(ownedAcctID: UInt64, child: Address, previousOwner: Address?, pendingOwner: Address)
```

Comment:
Personally, I am very much in favor of more explicit events. I believe the only other one to split up will be AccountUpdate events? Or was the intent to leave that as it currently is?

Reply:
My thought was we'd leave `AccountUpdated` and `OwnershipUpdated` as Child/OwnedAccount added/removed events. `ChildAccountPublished` and `OwnershipGranted` would signify Capability issuance on the publisher side. Think that makes sense?

Reply:
I think that makes sense!